cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -1,39 +1,32 @@
 package com.epam.cleancode.comments;
 
-public class MortgageInstallmentCalculator {
-
-    /**
-     *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
-     * @return monthly payment amount
-     */
-    public static double calculateMonthlyPayment(
-            int p, int t, double r) {
+class MortgageInstallmentCalculator {
+
+    private MortgageInstallmentCalculator() {}
+
+    static double calculateMonthlyPayment(
+            int principalAmount, int termOfMortgageYears, double rateOfInterest) {
 
         //cannot have negative loanAmount, term duration and rate of interest
```

Comment:
if you need comments it mean that code is not good. Please refactor it if you think that it's true or remove comments